### PR TITLE
green tick move to images folder

### PR DESCRIPTION
**Pull request by sujitxtendify**
https://www.screenpresso.com/=GvpTe

green tick img is moved to the image folder,

### CODE REVIEW

File: assets/images/green-tick-check.png
Comment (rosinghal) [inline assets/images/green-tick-check.png:1]: There is another file there with name green-tick, where is that being used?

Comment (sujitxtendify) [reply assets/images/green-tick-check.png:1]: this tick has use in checkbox content and that another file name green tick is used in the banner part .

https://www.screenpresso.com/=c2TGc this is a green-ticj-check.png
https://www.screenpresso.com/=X9qL this is a green-tick.png

**Review by rosinghal — APPROVED** [review]

**Comment by rosinghal** [comment]
@sujitxtendify please update the branch as my lint changes are causing conflict

**Comment by rosinghal** [comment]
@sujitxtendify lint test failing

**Review by rosinghal — APPROVED** [review]

**Review by Akhill2020 — APPROVED** [review]

**Comment by sujitxtendify** [comment]
> @sujitxtendify please update the branch as my lint changes are causing conflict

ok its done.

**Pull request merged**